backend/config/secrets.yml

```diff
@@ -5,6 +5,7 @@ development:
   qualtrics_api_key: 000000
   qualtrics_api_url: https://iad1.qualtrics.com/API/v3
   qualtrics_group_library: 'GR_7R5NeLAPtj22K46'
+  enclave_api_key: ec_1bG346BDBGEsG5Dck8H6sAiRBJ7Vt52Vwcno
```

Comment:
should these api keys be env-ified?

Reply:
it isn't a "real" api key, it's just for local testing, note that it's only in the dev & test blocks.  

in production it does read from ENV